app/src/main/java/com/marcelo/marvelheroes/domain/mapper/SortHeroesMapper.kt

```diff
@@ -0,0 +1,45 @@
+package com.marcelo.marvelheroes.domain.mapper
+
+import com.marcelo.marvelheroes.utils.constants.ASCENDING
+import com.marcelo.marvelheroes.utils.constants.DESCENDING
+import com.marcelo.marvelheroes.utils.constants.MODIFIED
+import com.marcelo.marvelheroes.utils.constants.NAME
+import com.marcelo.marvelheroes.utils.constants.ORDER_BY_MODIFIED_ASCENDING
+import com.marcelo.marvelheroes.utils.constants.ORDER_BY_MODIFIED_DESCENDING
+import com.marcelo.marvelheroes.utils.constants.ORDER_BY_NAME_ASCENDING
+import com.marcelo.marvelheroes.utils.constants.ORDER_BY_NAME_DESCENDING
+
+object SortHeroesMapper {
+    fun mapToPair(sorting: String): Pair<String, String> {
+        val orderBy = when (sorting) {
+            ORDER_BY_NAME_ASCENDING, ORDER_BY_NAME_DESCENDING -> NAME
+            ORDER_BY_MODIFIED_ASCENDING, ORDER_BY_MODIFIED_DESCENDING -> MODIFIED
```

Comment:
essa consulta vai ser feita no room ? Se for, pode sr mais facil jogar isso pro DAO